azure-pipelines.yml

```diff
@@ -69,12 +69,35 @@ jobs:
         python.version: '3.7'
       py38:
         python.version: '3.8'
+  variables:
+    PIP_CACHE_DIR: $(Pipeline.Workspace)/.pip
+    CCACHE_DIR: $(Pipeline.Workspace)/ccache
   steps:
   - task: UsePythonVersion@0
     inputs:
       versionSpec: '$(python.version)'
     displayName: 'Use Python $(python.version)'
 
+  - script: |
+      sudo apt update && sudo apt install -y ccache
+      echo "##vso[task.prependpath]/usr/lib/ccache"
+    displayName: Install ccache and update PATH to use linked versions of gcc, cc, etc
```

Comment:
where is the code which update the `PATH`? I see only a `echo` command here

Reply:
The echo command prepends it to the path.
This is copied from azure pipelines' documentation.